config/default/manager_auth_proxy_patch.yaml

```diff
@@ -1,54 +0,0 @@
-# This patch inject a sidecar container which is a HTTP proxy for the
```

Comment:
I guess this change related to previous commit.

If you get a chance to rebase again it will be nice moving it to the related commit, not a must.

Reply:
Why? It is the RBAC proxy, exactly what this commit is removing. What do I miss?